packages/api-key-service/src/functions/create.api-key.ts

```diff
@@ -4,43 +4,40 @@ import { validate } from 'class-validator'
 import { ApiKeyDto } from '@/dto'
 import { ApiKeyModel } from '@/models'
 import { createApiKey as create } from '@/services'
-import { authorised } from '@reapit/node-utils'
-import { connectSessionVerifyDecodeIdToken, LoginIdentity } from '@reapit/connect-session'
+import { connectSessionVerifyDecodeIdTokenWithPublicKeys, LoginIdentity } from '@reapit/connect-session'
+import publicKeys from './../../publicKeys.json'
 
 export const createApiKey = httpHandler<ApiKeyDto, ApiKeyModel>({
-  serialise: {
-    input: async (event): Promise<ApiKeyDto> => {
-      authorised(event)
-      let customer: LoginIdentity | undefined
+  handler: async ({ body, event }): Promise<ApiKeyModel> => {
+    let customer: LoginIdentity | undefined
 
-      try {
-        customer = await connectSessionVerifyDecodeIdToken(
-          event.headers['reapit-connect-token'] as string,
-          process.env.CONNECT_USER_POOL as string,
-        )
-      } catch (e) {
-        throw new UnauthorizedException(e.message)
+    try {
+      customer = await connectSessionVerifyDecodeIdTokenWithPublicKeys(
```

Comment:
Can this not just be `const customer: LoginIdentity | undefined`? Doesn't look like you use it outside of the try block... unless I am missing something?

Reply:
yea I'm using it for customer.developerId
https://github.com/reapit/foundations/pull/4251/files/90a4748496a8338ee6c4316334b54b90022502ee#diff-e440ae60e55a2240bac5f822c3b7c132462405d68292bbf4b4d3392d3d2cc82fR31